pkg/webhook/parser_integration_test.go

```diff
@@ -175,6 +175,55 @@ var _ = Describe("Webhook Generation From Parsing to CustomResourceDefinition",
 		assertSame(actualMutating, expectedMutating)
 		assertSame(actualValidating, expectedValidating)
 	})
+
+	It("should generate the ordered webhook definitions", func() {
```

Comment:
Shame we aren't using ginkgo V2 as you could use `MustPassRepeatedly(10)` if we were, maybe a future improvement